File: Backend/Backend/Models/EstudianteModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Backend.Models
{
    public class EstudianteModel
    {
        public int idEstudiante { get; set; }
        public int nombreEstudiante { get; set; }
        public int apellidoEstudiante { get; set; }
        public int fechaNacimientoEstudiante { get; set; }
        public int direccionEstudiante { get; set; }
        public int telefonoEstudiante { get; set; }
        public int identificacionEstudiante { get; set; }
        public AsignaturaModel[] asignaturaEstudiante { get; set; }

    }
}

File: FrontEnd/src/app/Models/estudiante.model.ts
import { AsignaturaModel } from "./asignatura.model";

export class EstudianteModel {
  public idEstudiante: number;
  public nombreEstudiante: number;
  public apellidoEstudiante: number;
  public fechaNacimientoEstudiante: number;
  public direccionEstudiante: number;
  public telefonoEstudiante: number;
  public identificacionEstudiante: number;
  public asignaturaEstudiante: AsignaturaModel[]=[];
}
